Lib/test/test_typing.py

```diff
@@ -4930,6 +4930,8 @@ def test_args_kwargs(self):
         self.assertIsInstance(P.kwargs, ParamSpecKwargs)
         self.assertIs(P.args.__origin__, P)
         self.assertIs(P.kwargs.__origin__, P)
+        self.assertEqual(P.args, P.args)
```

Comment:
We totally need a case, when they are not equal.
For example, `P1.args` and `P2.args`?

Reply:
Good point. I added tests for this.